classes/class-loader.php

```diff
@@ -346,6 +346,8 @@ public function enqueue_public_assets() {
 			],
 			$css_blocks_creation
 		);
+
+		wp_enqueue_script( 'media', P4GBKS_PLUGIN_URL . 'public/js/media.js', [ 'jquery' ], '0.1', true );
```

Comment:
You are deleting this script in this PR. So do we still need it?

Reply:
@comzeradd Good point. I removed it